test/evp_extra_test2.c

```diff
@@ -547,6 +506,47 @@ static int test_new_keytype(void)
     return ret;
 }
 
+#ifndef OPENSSL_NO_EC
+
+static int test_ec_d2i_i2d_pubkey(void)
+{
+    int ret = 0;
+    FILE *fp = NULL;
+    EVP_PKEY *key = NULL, *outkey = NULL;
+    static const char *filename = "pubkey.der";
+
+    if (!TEST_ptr(fp = fopen(filename, "wb"))
```

Comment:
I know this is just transported from the old location, but does it make sense to use BIO_new_file here instead of fopen directly?  So that we pick up the special cases for Win32 and DGJPP?